cmd/flags.go

```diff
@@ -241,7 +241,8 @@ var (
 	// GrpcMaxCallRecvMsgSizeFlag defines the max call message size for GRPC
 	GrpcMaxCallRecvMsgSizeFlag = &cli.IntFlag{
 		Name: "grpc-max-msg-size",
-		Usage: `Integer to define max receive message call size (in bytes).
+		Usage: `WARNING: gRPC API is being deprecated in favour of REST API.
```

Comment:
maybe we can have a constant for this , I'm iffy on the wording because I still think it will be a while before we can actually switch to REST, is there a way to soften the wording because it feels like it will impact users very soon.